frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -101,14 +101,14 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
 
   return (
     <Modal
-      title="Deploy model"
+      title={!editInfo ? 'Deploy model' : 'Edit model'}
```

Comment:
small nit (here and a few other places), this would be one less logical step in a developer's head if you reversed the logic/order.

```suggestion
      title={editInfo ?  'Edit model' : 'Deploy model'}
```